src/main/java/practice/seema/Program1.java

```diff
@@ -0,0 +1,30 @@
+package practice.seema;
+
+import java.util.Scanner;
+
+//Print the fibbonassci series
+public class Program1 {
+   public static void main(String args[]){
+
+
+       Scanner in = new Scanner(System.in);  // Reading from System.in
+       System.out.println("Enter a number: ");
+       int number = in.nextInt();
+       int firstTerm=0;
+       int secondTerm=1;
+       int thirdTerm;
+       System.out.println(firstTerm);
+       System.out.println(secondTerm);
+       int i=3;
```

Comment:
here you should take i=2;bcoz 0 &  1 already taken so next is 2 not 3

Reply:
i is not for print the value .as I print first and second term, so I am stared the loop from third position